app/controllers/farms_controller.rb

```diff
@@ -12,7 +12,7 @@ def new
   end
 
   def create
-    Farm.create(farm_params)
+    Farm.create!(farm_params)
```

Comment:
does this need to throw an error in the server environment? I thought the bang was mostly for testing.